publish-service/src/main/java/com/ericsson/eiffel/remrem/publish/controller/ProducerController.java

```diff
@@ -87,6 +95,18 @@ public void setRestTemplate(RestTemplate restTemplate) {
         this.restTemplate = restTemplate;
     }
 
+    public void getUserName() {
```

Comment:
The name getUserName() indicates that user name is returned and it might be confusing. Please, rename to something more appropriate, e.g. logUserName().